web-ui/src/main/resources/catalog/locales/fr-gnui.json

```diff
@@ -0,0 +1,27 @@
+{
+  "facets.block.title.OrgForResource": "Organisation",
```

Comment:
could those field be removed from gn-ui i18n files ? couldn't we get them from somewhere in the backend ?

Reply:
In the long run, yes that sounds the way to go. We have a database API for storing custom translations https://github.com/geonetwork/core-geonetwork/pull/4823 but we lack a UI for people to manage them in the admin - we missed the opportunity to have that funded by geocat.ch ... it will come at some point. Then indeed as user can customize facet from the admin, they have to also have the possibility to manage the label of the facet. So it totally makes sense; still a bit of work before that move.

Reply:
But facet block title are iso19139 element names.. so they are translated already somewhere in the backend, it's not custom translation, is it ?

Reply:
Not always, facet are not only related to schema - some of them are related to Index structure (those one are common), some other are dynamic and based on schema element name eg. cl_.... and some facet are custom eg. filter based so indeed we can maybe improve at some point with translations for global index fields, then for schema we are not going to load all schema translations only for 2 facet labels ... 

Some are also related to thesaurus title - and quite often end user does not use the thesaurus title as is ...  so something to think about.